src/main/resources/json/schema/update_payload_schema.json

```diff
@@ -11,7 +11,7 @@
     },
     "state": {
       "description": "The state node containing the desired, reported, and delta shadow state.",
-      "type": "object",
+      "type": ["object", "null"],
```

Comment:
so we're considering these values as valid now, which is fine, but are we actually handling the values correctly?

What is our behavior after validating the data?

Reply:
yeah realized this after more testing, currently reviewing/updating merging logic to handle null/empty state